src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -0,0 +1,208 @@
+      mappedIdsSet.size,
+      'a schema may have been added to vets-json-schema/dist/schemas',
+    );
+    expect(allFormIdsSet).to.deep.equal(mappedIdsSet);
+    expect(includedFormIds).to.deep.equal(
+      reformattedIds,
+      'possible missing formId property in a formConfig',
+    );
+  });
+
+  configs.forEach(form => {
+    describe(`${form.formId}:`, () => {
+      describe('migrations:', () => {
+        const { migrations } = form;
+        if (migrations || form.version > 0) {
+          it('should have a length equal to the version number', () => {
+            expect(migrations.length).to.equal(form.version);
+          });
+          it('should be typeof array', () => {
+            expect(migrations).to.be.an('array');
+          });
+          it('should be array of functions', () => {
+            expect(
+              migrations.every(migration => typeof migration === 'function'),
+            ).to.be.true;
+          });
+        }
+      });
+
+      it('should have chapters object', () => {
+        expect(form.chapters).to.be.an('object');
+      });
+
+      it('should have defaultDefinitions object', () => {
+        expect(form.defaultDefinitions).to.be.an('object');
+      });
+
+      it('should have introduction function', () => {
+        expect(form.introduction).to.be.an('function');
+      });
+
+      it('should have prefillEnabled boolean', () => {
+        expect(form.prefillEnabled).to.be.a('boolean');
+      });
+
+      if (form.prefillTransformer) {
+        it('should have prefillTransformer function', () => {
+          expect(form.prefillTransformer).to.be.a('function');
+        });
+      }
+
+      it('should have trackingPrefix', () => {
+        expect(form.trackingPrefix).to.be.a('string');
+      });
+
+      it('should have title', () => {
+        expect(form.title).to.be.a('string');
+      });
+
+      if (form.subTitle) {
+        it('should have subTitle', () => {
+          expect(form.subTitle).to.be.a('string');
+        });
+      }
+
+      it('should have urlPrefix', () => {
+        expect(form.urlPrefix).to.be.a('string');
+      });
+
+      if (form.submitUrl) {
+        it('should have submitUrl', () => {
+          expect(form.submitUrl).to.be.a('string');
+        });
+      }
+
+      if (form.submit) {
+        it('should have submit', () => {
+          expect(form.submit).to.be.a('function');
+        });
+      }
+
+      it('should have savedFormMessages', () => {
+        expect(form.savedFormMessages).to.be.a('object');
+      });
+
+      if (form.transformForSubmit) {
+        it('should have transformForSubmit', () => {
+          expect(form.transformForSubmit).to.be.a('function');
+        });
+      }
+
+      it('should have confirmation', () => {
+        expect(form.confirmation).to.be.a('function');
+      });
+
+      it('should have preSubmitInfo', () => {
+        expect(form.preSubmitInfo).to.be.a('object');
```

Comment:
:thinking: I'm not sure what this is. Do all forms need this?

Reply:
The default is https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/preSubmitInfo.js

<img width="649" alt="Screen Shot 2019-08-21 at 10 41 55 AM" src="https://user-images.githubusercontent.com/1094999/63441998-55cc3080-c400-11e9-874a-3e648ee6d544.png">

Reply:
if it's not required for all forms i can wrap the test in a if property exists check

Reply:
Okay, yeah. If we supply a default and don't require this to be in the `formConfig`, my vote is for wrapping it in a conditional. :+1: